app_dart/lib/src/request_handling/cache_request_handler.dart

```diff
@@ -0,0 +1,70 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:typed_data';
+
+import 'package:meta/meta.dart';
+import 'package:neat_cache/neat_cache.dart';
+import 'package:pedantic/pedantic.dart';
+
+import '../datastore/cocoon_config.dart';
+import '../request_handling/request_handler.dart';
+import 'body.dart';
+
+/// A [RequestHandler] for serving cached responses.
+///
+/// High traffic endpoints that have responses that do not change
+/// based on request are good for caching. Additionally, saves
+/// reading from Datastore which is expensive both timewise and monetarily.
+@immutable
+class CachedRequestHandler<T extends Body> extends RequestHandler<T> {
```

Comment:
_Super-nit_: The class name is `CachedRequestHandler`, but the file name is `cache_request_handler.dart`.  The two should match.

Reply:
Good catch! Renamed class to `CacheRequestHandler`.